README.md

```diff
@@ -219,6 +219,17 @@ import errorHandler from './errorHandlerUtility';
 
 **Q: Are private source maps supported?** A: No, see [#4](https://github.com/GoogleCloudPlatform/stackdriver-errors-js/issues/4)
 
+**Q: Can I use this without an API key?** A: Yes, see below
```

Comment:
Can you add that the custom URL must implement the `report` API endpoint: https://cloud.google.com/error-reporting/reference/rest/v1beta1/projects.events/report